lib/strategies/android-chrome.js

```diff
@@ -9,6 +10,26 @@ function AndroidChromeStrategy(options) {
 AndroidChromeStrategy.prototype = Object.create(BaseStrategy.prototype);
 AndroidChromeStrategy.prototype.constructor = AndroidChromeStrategy;
 
+BaseStrategy.prototype._attachListeners = function () {
```

Comment:
Could we shorten this to:

```js
var self = this;
var keyUpAndInputHandler = self._reformatInput.bind(self);

function keyDownAndKeyPressHandler(event) {
  if (keyCannotMutateValue(event)) { return; }
  self._unformatInput(event);
}

self.inputElement.addEventListener('keydown', keyDownAndKeyPressHandler);
self.inputElement.addEventListener('keypress', keyDownAndKeyPressHandler);

self.inputElement.addEventListener('keyup', keyUpAndInputHandler);
self.inputElement.addEventListener('input', keyUpAndInputHandler);

self.inputElement.addEventListener('paste', self.pasteEventHandler.bind(self));
```

It's a little less repetitive, though it may not be as clear. Thoughts?